Add sub-page for open hardware

Thesaleem opened the pull request:
In this PR, I worked on the subpage for open hardware. 

- I gave a short summary and then added some resources withe the youtube iframe for the video. 
- I  translated the content to Arabic 
- Easy accessibility and scrolling  to the contents of the homepage from the navbar in the subpage. 

I think the content of the website can be improved on by giving updates, news or upcoming events in different respective research fields.

BatoolMM commented:
Thank you @Thesaleem  for your work. A few comments about the PR.

- Can you change the outline of the text and make it better organised (bullet points) - similar to this PR #38.
- I love how you managed to get the right alignment for Arabic.
- Add where the text is cited from.
- Add where the image is taken from
- Add a few sentences about the video. 
> This video is part of [Open Life Sciences Programe](https://openlifesci.org/) and the subtitles are available in Arabic and English. The OLS program is for people interested in applying open principles in their work and becoming Open Science ambassadors in their communities.
You may also add a reference for the video:
> Please cite: Malvika Sharan, Emmy Tsang, Yo Yehudi, Bérénice Batut, Carlos Martinez, Helena Rasche, Hao Ye, Esther Plomp, Andre Maia Chagas, Georgia Aitkenhead, Suzanne lwai, Otis Smith, Emma Anne Harris, Thomas Mboa, Alex Holinski, Zulidyana Rusnalasari, Nelsy Mtsweni, Rowland Mosbergen, Iratxe Puebla, … Anelda van der Walt. (2021, September 19). Mini Talks/Videos about Open Science from OLS-3 Cohort (v.1.1). Zenodo. https://doi.org/10.5281/zenodo.5515763
- Since open educational resources have been submitted with multiple PR. Would you like to try "Citizen Science". You can re-use materials from [the Turing Way here](https://the-turing-way.netlify.app/reproducible-research/open/open-scholarship.html?highlight=citizen%20science) with cittaion.

Thank you again for your hard work and looking for more contributions!

Thesaleem commented:
Hi @BatoolMM! Thanks for the feedback. To be clear, do you mean I should add the last sentence which is; (This video is part of Open Life Sciences Programe and the subtitles are available... ) as the video text? And I think it might be best to add the video reference to the other references I listed.

BatoolMM commented:
You have to add it as a reference on the page - needs to show credit for any work taken from other projects or websites.

Thesaleem commented:
Hi @BatoolMM! I have made an update to the PR. I have updated the texts in bullet points, added the citations and updated the references for the English and Arabic pages. I'm looking forward to your review. Thanks!

alswajiab commented:
Thank you, @Thesaleem, for your contribution. I have some comments.
- Have a way to get back to the main page.
-  This image[](https://deploy-preview-36--oscksa.netlify.app/assets/img/oshw-logo.png) caption is relatively big; can you make it smaller?
- There’s no caption for this image[](https://deploy-preview-36--oscksa.netlify.app/assets/img/BookShelf.jpg) (I suggest removing it).

Thesaleem commented:
Thank you @alswajiab for the kind comments. Do you suggest I add a “back button” to the page as I made sure the home link on the nav bar navigates one back to the main page.

alswajiab commented:
> Thank you @alswajiab for the kind comments. Do you suggest I add a “back button” to the page as I made sure the home link on the nav bar navigates one back to the main page.

I suggest something like `Home / Open Hardware`.

Thank you @Thesaleem

Thesaleem commented:
> > Thank you @alswajiab for the kind comments. Do you suggest I add a “back button” to the page as I made sure the home link on the nav bar navigates one back to the main page.
> 
> I suggest something like `Home / Open Hardware`.
> 
> Thank you @Thesaleem

@alswajiab Got that! I'm on it.

Thesaleem commented:
@alswajiab PR has been updated. Kindly review it, thank you.

alswajiab commented:
Amazing @Thesaleem, I have one comment on your PR:
- [ ] Add Arabic translation to the footer on the Arabic page.

Thesaleem commented:
> Amazing @Thesaleem, I have one comment on your PR:
> 
> * [ ]  Add Arabic translation to the footer on the Arabic page.

Updated already! @alswajiab

Thesaleem commented:
Hi @BatoolMM! I was wondering if you’ve had time to review this PR.
